text/0032-minimal-relay.md

```diff
@@ -0,0 +1,196 @@
+# RFC-0032: Minimal Relay
+
+|                 |                                                                               |
+| --------------- | ----------------------------------------------------------------------------- |
+| **Start Date**  | 20 September 2023                                                             |
+| **Description** | Proposal to minimise Relay Chain functionality.                               |
+| **Authors**     | Joe Petrowski, Gavin Wood                                                     |
+
+## Summary
+
+The Relay Chain contains most of the core logic for the Polkadot network. While this was necessary
+prior to the launch of parachains and development of XCM, most of this logic can exist in
+parachains. This is a proposal to migrate several subsystems into system parachains.
+
+## Motivation
+
+Polkadot's scaling approach allows many distinct state machines (known generally as parachains) to
+operate with common guarantees about the validity and security of their state transitions. Polkadot
+provides these common guarantees by executing the state transitions on a strict subset (a backing
+group) of the Relay Chain's validator set.
+
+However, state transitions on the Relay Chain need to be executed by _all_ validators. If any of
+those state transitions can occur on parachains, then the resources of the complement of a single
+backing group could be used to offer more cores. As in, they could be offering more coretime (a.k.a.
+blockspace) to the network.
+
+By minimising state transition logic on the Relay Chain by migrating it into "system chains" -- a
+set of parachains that, with the Relay Chain, make up the Polkadot protocol -- the Polkadot
+Ubiquitous Computer can maximise its primary offering: secure blockspace.
+
+## Stakeholders
+
+- Parachains that interact with affected logic on the Relay Chain;
+- Core protocol and XCM format developers;
+- Tooling, block explorer, and UI developers.
+
+## Explanation
+
+The following pallets and subsystems are good candidates to migrate from the Relay Chain:
+
+- Identity
+- Balances
+- Staking
```

Comment:
Small note that the Staking and EPM pallet would need significant refactor and may be almost a complete rewrite in case of EPM before being ready for Staking Chain. As it stands currently, they are not very PoV friendly.